coordinator/src/orderbook/trading.rs

```diff
@@ -16,6 +16,11 @@ pub fn match_order(order: Order, all_orders: Vec<Order>) -> Result<Vec<MatchPara
         return Ok(vec![]);
     }
 
+    let all_orders = all_orders
```

Comment:
I still think this should not be called `all_orders`, because it is not all orders.

Reply:
happy to do so :)